doc/releases/release-notes-3.2.rst

```diff
@@ -223,27 +223,95 @@ Bluetooth
+  * Added ASCS Metadata verification support
+  * Started exposing broadcast sink advertising data to the application
+  * Added support for unicast server start, reconfigure, release, disable and
+    metadata
+  * Added support for multi-CIS
+  * Implemented HAS client support for preset switching
+  * Added support for setting vendor-specific non-HCI data paths for audio
+    streams
+
 * Direction Finding
 
+  * Added support for selectable IQ samples conversion to 8-bit
+  * Added support for VS IQ sample reports in ``int16_t`` format
+
 * Host
 
-  * Added a new callback :c:func:`rpa_expired` in the struct :c:struct:`bt_le_ext_adv_cb`
-    to enable synchronization of the advertising payload updates with the Resolvable Private
-    Address (RPA) rotations when the :kconfig:option:`CONFIG_BT_PRIVACY` is enabled.
-  * Added a new :c:func:`bt_le_set_rpa_timeout()` API call to dynamically change the
-    the Resolvable Private Address (RPA) timeout when the :kconfig:option:`CONFIG_BT_RPA_TIMEOUT_DYNAMIC`
-    is enabled.
-  * Added :c:func:`bt_conn_auth_cb_overlay` to overlay authentication callbacks for a Bluetooth LE connection.
-  * Removed ``CONFIG_BT_HCI_ECC_STACK_SIZE``.
-    The Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`) is used for processing ECC commands instead of the dedicated thread.
-  * :c:func:`bt_conn_get_security` and `bt_conn_enc_key_size` now take a ``const struct bt_conn*`` argument.
+  * Added support for LE Secure Connections permission checking
+  * Added support for Multiple Variable Length Read procedure without EATT
+  * Added a new callback :c:func:`rpa_expired` in the struct
+    :c:struct:`bt_le_ext_adv_cb` to enable synchronization of the advertising
+    payload updates with the Resolvable Private Address (RPA) rotations when
+    the :kconfig:option:`CONFIG_BT_PRIVACY` is enabled
+  * Added a new :c:func:`bt_le_set_rpa_timeout()` API call to dynamically change
+    the the Resolvable Private Address (RPA) timeout when the
+    :kconfig:option:`CONFIG_BT_RPA_TIMEOUT_DYNAMIC` is enabled
+  * Added :c:func:`bt_conn_auth_cb_overlay` to overlay authentication callbacks
+    for a Bluetooth LE connection
+  * Removed ``CONFIG_BT_HCI_ECC_STACK_SIZE``. A new Bluetooth long workqueue
+    (:kconfig:option:`CONFIG_BT_LONG_WQ`) is used for processing ECC commands
+    instead of the former dedicated thread
+  * :c:func:`bt_conn_get_security` and :c:func:`bt_conn_enc_key_size` now take
+    a ``const struct bt_conn*`` argument
+  * The handling of GATT multiple notifications has been rewritten, and is now
+    only to be used as a low-level API
+  * Added support for GATT CCCs in arbitrary locations as a client
+  * Extended the ``bt_conn_info`` structure with security information
+  * Added a new :kconfig:option:`CONFIG_BT_PRIVACY_RANDOMIZE_IR` that prevents
+    the Host from using Controller-provided identity roots
+  * Added support for GATT over EATT
+  * Implemented the Immediate Alert Client
```

Comment:
I think there's also the channel selection option for EATT. Don't know if we should mention it here, what do you think @hermabe ?

Reply:
>  * Added support for GATT over EATT

Is it not mentioned on the line above?

However, I think I messed up when merging the note in the release note and it was added to the 3.1 release notes. The PR was opened before 3.1 was released, but merged afterwards.
https://github.com/zephyrproject-rtos/zephyr/blob/2cea5850e2cd7587858276da9d8c84ee90568824/doc/releases/release-notes-3.1.rst?plain=1#L28-L30

We should move it to the 3.2 release notes.

Reply:
I added it to the 3.2 release notes. Left it in the 3.1 ones to avoid confusion.